example.config.json

```diff
@@ -0,0 +1,62 @@
+{
+    "secrets": {
+        "s3": {// for storage=s3
```

Comment:
I'm okay with changing the format of the config file, but JSON does not support comments and I think that's important for documenting a file with this much in it. It's also annoying to have to remove all of the comments from the example if you're using that as a base for your own config.json file. I'd suggest using a format that supports comments for this (either Python itself or maybe YAML or something like that).

Reply:
good point, rewrote it to use YAML and config.yaml instead